_posts/2017-12-28-utah-ng911-gis-database.md

```diff
@@ -0,0 +1,46 @@
+
+Today, thanks to the willingness and cooperation of agencies within the State of Utah, we are excited to announce the first release of the Utah NG911 GIS database. This database is built on the National Emergency Number Association’s (NENA) Standard for NG9-1-1 GIS Data Model, in its current form. Schema-wise, AGRC will keep current with this standard and update the Utah NG911 GIS database if and when NENA makes changes to the specifications. NENA hopes to finalize this standard soon.  Data-wise, AGRC will update the Utah NG911 GIS database on a monthly cycle - in conjunction with our road update cycle. However, AGRC can, and will, make updates more frequently when the data suggests.
+
+![NG911 Datasets]({{ "/images/utah-ng911-database.png" | prepend: site.baseurl }})
+
+## Looking Ahead
+
+In a true NG911 system, GIS becomes front and center. This puts a lot of responsibility on authoritative GIS editors at the local, regional (PSAP), and State levels. In 2018, AGRC looks to make further progress on providing a feedback mechanism for all approved authoritative data editors (such as local jurisdictions and PSAPs) where they can submit data modifications in a mapping editing environment and submit them to us so we can push them into production within the established time period. This will ensure the availability of highly accurate, near real time data to the State’s PSAPs and 911 community (telecoms?).  This is a really important aspect to AGRC because this mechanism can save lives!
+
+AGRC also has plans to geocode the MSAG tables against the Utah NG911 GIS database, looking for MSAG address that do not geocode. Our goal is to ensure that all existing, valid MSAG address are represented in the GIS database. This will speed up the transition to NG911 and GIS-based call routing, while undermining any reliance on the MSAG tables.
+
+We encourage you download the Utah NG911 GIS Database and take a look at it. We are strong believers that the more data is used and viewed, the better it gets. If nothing else, make sure your home and/or business is represented as a geocodable address in both the “RoadCenterlines” and “AddressPoints” datasets. These are the datasets that will ensure proper 911 response to your location. If you don’t have the needed tools and software to interact with the NG911 dataset, please take a moment to use our mapping website where you can validate your home and/or business address using the same underlying datasets. These datasets are only as good as we collectively make them, so do your part and validate.
+
+Questions or feedback can be directed to Greg Bunce at gbunce@utah.gov.    
```

Comment:
please dont' use your specific email. Please use the [contacts.yml](https://github.com/agrc/gis.utah.gov#contact-information) file.